src/components/menu/menu-item/menu-item.style.js

```diff
@@ -56,6 +56,8 @@ const StyledMenuItemWrapper = styled.a`
             ${maxWidth &&
             css`
               box-sizing: border-box;
+              border-bottom-left-radius: 6px;
```

Comment:
@harpalsingh is the focus styling correct or should it be like the items in `ActionPopover`?

![image](https://user-images.githubusercontent.com/44157880/228317110-ea5f2ce2-1980-4060-bf1d-1e96b493e198.png)